### DISCUSSION TITLE
Table 1, Practical 3

### DISCUSSION THREAD

**Discussion by Jonathan-Lee-UofG** [IBAHCM/RPiR#438] [Old]
Github username: Jonathan-Lee-QMBCE
Repository link: https://github.com/IBAHCM/JonathanLeeQmbceSeries3.git

**Comment by 2700638AichiMkunde** [comment]
https://github.com/IBAHCM/AichiMkundeseries03.git

**Comment by nadiareeve** [comment]
https://github.com/IBAHCM/nadiareeveSeries3.git

**Reply by 2700638AichiMkunde** [reply]
May you send an invitation please?

**Reply by 2700638AichiMkunde** [reply]
> https://github.com/IBAHCM/nadiareeveSeries3.git

I can access your repository now. 
Thank you

**Comment by 2700638AichiMkunde** [comment]
> Github username: Jonathan-Lee-QMBCE Repository link: https://github.com/IBAHCM/JonathanLeeQmbceSeries3.git

I can't access your codes is that ok, or you need to send an invitation

**Reply by 2700638AichiMkunde** [reply]
> > Github username: Jonathan-Lee-QMBCE Repository link: https://github.com/IBAHCM/JonathanLeeQmbceSeries3.git
> 
> I can't access your codes is that ok, or you need to send an invitation

I can access your repository now.
Thank you.

**Comment by 2700638AichiMkunde** [comment]
Hello @Jonathan-Lee-QMBCE @nadiareeve 

I had like feedback I'm supposed to comment on your repositories after checking your code.

I hope, I did it as it is supposed?

Or if I haven't please let me know.

I also opened for you to access my repository.

Thanks
